saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -65,6 +65,14 @@ class OrderLineCreateInput(OrderLineInput):
             "by skipping the matching logic. " + ADDED_IN_36
         ),
     )
+    price = PositiveDecimal(
+        required=False,
+        description=(
+            "Custom price of the item."
+            "When the line with the same variant "
+            "will be provided multiple times, the last price will be used."
```

Comment:
You should add `ADDED_IN_314` + (probably) `PREVIEW_FEATURE` to all new fields that you're adding